docs/using/recipes.md

```diff
@@ -568,3 +568,20 @@ docker run -it --rm \
     -v /tmp/.X11-unix:/tmp/.X11-unix \
     jupyter/minimal-notebook
 ```
+
+## Add ijavascript kernel to container
+
+The example below is a Dockerfile to install the ijavascript kernel.
+
+```dockerfile
+# use one of the jupyter docker stacks images
+FROM jupyter/scipy-notebook:85f615d5cafa
```

Comment:
Can we use a date tag here instead? That way, we can know that this worked at a certain point in time very easily and track down whats changed since a lot better.

Reply:
We definitely can and I will do it in a separate PR.
There are many places, where this concrete tag is used.